src/main/io/osd.c

```diff
@@ -435,6 +435,14 @@ static void osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_REM_TIM_EST:
+	if (!getMAhDrawn())
```

Comment:
Please don't use double negatives in `if` statements.
Should be:
```
if (getMAhDrawn()) {
    tfp_sprintf(buff, "%4dS", (int)((osdConfig()->cap_alarm - getMAhDrawn()) * ((float)flyTime / 1000000) / getMAhDrawn()));
} else {
    // FIXME add 's' to font
    tfp_sprintf(buff, "NA S");
}
```

Note also braces should be added to all `if` statements.